clustohttp.py

```diff
@@ -335,6 +335,12 @@ def del_attr(self, key):
         Returns EntityProxy of the pool whose attributes removed
         '''
         path = "delattr?" + "key=" + key
+        if subkey:
+            path += "&subkey=" + subkey
```

Comment:
dont you need to sanitize the strings? eg how are you going to handle a value with spaces eg Pool(app=foo, bar=...)